src/osctable.h

```diff
@@ -0,0 +1,70 @@
+#ifndef __OSCTABLE_H__
+#define __OSCTABLE_H__
+
+/**
+ * \file osctable.h
+ *
+ * Handlers for oscillation tables
+ */
+
+
+class OscTable {
```

Comment:
It doesn't look like this class is actually instantiated anywhere, but that the constructor is used like a function around https://github.com/mastbaum/sxmc/pull/6/files#diff-f1adcc313a7ed00a25e1d3511c382516R176. If that's the case, could it just be a function?

Reply:
Rewrote this as a function and moved it.  Structure now more like ttree io.